app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -13,27 +12,58 @@ def self.call(*args)
       def call
         # TODO: [@thepracticaldev/delightful] Move this check into the rake task logic once it has been implemented.
         return unless user.welcome_notifications
-        return if commented_on_welcome_thread? || received_notification?
+
+        send_welcome_notification
```

Comment:
i am just now wondering if we should `return` and short-circuit out of these notifications 🤔 

is it possible right now for both `send_welcome_notification` and `send_authentication_notification` to run? could someone run this generator and send multiple notifications to the same user at once...?

Reply:
Yes that's possible right now I even wrote a spec (`::cal`) to make sure it's possible 😓 . I'll make that change